### ISSUE TITLE
examples doesn't build at HEAD (`2cedce1`)

### ISSUE THREAD

**Issue by bileschi** [nannou-org/nannou#711]
Ran the command (see below) on a clean checkout of nannou

Context:
*  macos ver 11.2.1
*  cargo 1.50.0 (f04e7fab7 2021-02-04)
*  rustc 1.50.0 (cb75ad5db 2021-02-10)

```
$ cargo run --release --example draw
    Updating crates.io index
   Compiling libc v0.2.86
   Compiling sample v0.9.1
   Compiling rand_core v0.3.1
   Compiling thread_local v0.3.6
   Compiling rayon-core v1.9.0
   Compiling clang-sys v0.21.2
   Compiling libloading v0.4.3
   Compiling regex v0.2.11
   Compiling regex-syntax v0.5.6
   Compiling rand_chacha v0.1.1
   Compiling rand_pcg v0.1.2
   Compiling deflate v0.7.20
   Compiling inflate v0.3.4
   Compiling rand v0.6.5
   Compiling deflate v0.8.6
   Compiling miniz_oxide v0.3.7
   Compiling maybe-uninit v2.0.0
   Compiling typenum v1.12.0
   Compiling proc-macro2 v0.2.3
error[E0282]: type annotations needed
  --> /Users/bileschi/.cargo/registry/src/github.com-1ecc6299db9ec823/sample-0.9.1/src/window.rs:73:41
   |
73 |         let v = phase.to_float_sample().to_sample() * PI_2;
   |                                         ^^^^^^^^^ cannot infer type for type parameter `S` declared on the associated function `to_sample`
   |
help: consider specifying the type argument in the method call
   |
73 |         let v = phase.to_float_sample().to_sample::<S>() * PI_2;
   |                                                  ^^^^^

error: aborting due to previous error

For more information about this error, try `rustc --explain E0282`.
error: could not compile `sample`

To learn more, run the command again with --verbose.
warning: build failed, waiting for other jobs to finish...
error: build failed
```

**Comment by EliKor** [comment]
I believe this is same as #703 and #704.

**Comment by mitchmindtree** [comment]
Thanks for the report! Closed via #665.
